Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -41,11 +41,6 @@ class PlaceCarouselViewController: UIViewController {
             // TODO: how do we make sure the user wasn't interacting?
             headerView.numberOfPlacesLabel.text = "\(places.count) place" + (places.count != 1 ? "s" : "")
             placeCarousel.refresh()
-
-            if oldValue.isEmpty,
```

Comment:
I was experiencing some weird timing/navigation issues on my transition animation branch with this here so I moved it out of the setter and call it explicitly after we set places in `didReceivedPlaces` below